src/uk/ac/open/tabapplet/TabApplet.java

```diff
@@ -280,6 +282,23 @@ private void ditchFocus(boolean forward)
 		evalJS(js);
 	}
 
+	/**
+	 * To check if VR is paused
+	 * @param ispause boolean 
+	 **/
+	public void isPause(boolean isPause)
```

Comment:
The TabApplet class is supposed to be a generic one to deal with all applets you can tab around... Could this code be moved into the more specific applet class just for the voice recorder, I guess it is VoiceRecorderApplet or something?

Reply:
Moved to EmbeddedRecorderApplet